tests/setup.js

```diff
@@ -1,5 +1,5 @@
-import enzyme, { shallow, mount } from "enzyme";
-import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
+const Adapter = require("@cfaester/enzyme-adapter-react-18").default;
```

Comment:
Let's make issue to remove this deprecated library.

Reply:
created [issue](https://github.com/kbss-cvut/record-manager-ui/issues/128) for this